README.md

```diff
@@ -16,9 +17,24 @@ We want to build proteus in a very extensible way, so every step of the generati
 go get -v github.com/src-d/proteus/...
 ```
 
+### Requirements
+
+You need two requirements for the full process.
```

Comment:
I see it quite strange; I'd say:
- "_there are two requirements_", or
- "_you need to fulfill two requirements_"